.gitignore

```diff
@@ -1,11 +1,11 @@
 deno.d.ts
 dist
-node_tests/*
-!node_tests/node_tests.ts
-!node_tests/tsconfig.json
-!node_tests/package.json
-!node_tests/README.md
-!node_tests/fixtures
+.github/node_tests/*
```

Comment:
What's the reasoning behind this change? In addition to the CI aspect I thought that the node tests were also a code example of how you can use ts_serialize in a typescript context, so it made sense to have them as part of the docs.

On the other hand node usage is covered in the README, so I'm fine with this change

Reply:
basically from conversations, it was trying to be both examples and tests, but not doing a good job of either. This move is to remove it from the root so the docs/README is used. With the changes to the readme now this is only used on `github` so it made sense to move the folder. 

TBH, [the deno -> node conversation](https://github.com/denoland/deno/discussions/9569) has probably moved forward from when we last looked and there might exist a solution that allows us to run the deno tests within a node env.

Reply:
Sounds good, I will create a discussion to see if we can find a better way to transpiling into node